api/api/migrations/0056_rename_mature_to_sensitive.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.9 on 2024-02-08 18:26
+
+import api.models.media
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0055_alter_matureaudio_table_alter_matureimage_table'),
+    ]
+
+    operations = [
+        migrations.RenameModel('MatureImage', 'SensitiveImage'),
+        migrations.RenameModel('MatureAudio', 'SensitiveAudio'),
+        migrations.AlterField(
+            model_name='sensitiveaudio',
+            name='media_obj',
+            field=models.OneToOneField(db_column='identifier', db_constraint=False, help_text='The reference to the sensitive audio.', on_delete=django.db.models.deletion.DO_NOTHING, primary_key=True, related_name='sensitive_audio', serialize=False, to='api.audio', to_field='identifier'),
```

Comment:
All of these fields are matched on `identifier`, which is carried over from the catalog. Since the API tables are rebuilt from the catalog, I think having the data persist in the sensitive tables even if it's somehow removed from the API table, makes sense in case the record for that identifier reappears from the catalog. Normally I think a `CASCADE` would make sense, but based on how we construct the tables I think it's safer to take no actions here.

Reply:
Ah, good point! It's the cumbersome of redoing the media tables 😕 It makes difficult to actually honor the DB constraints. 

Thanks for clarifying!

Reply:
No problem! Is there anything else on this blocking your approval of the PR?